### PR TITLE
Integrated Data Binding API

### PR DESCRIPTION
**Pull request by phanirajabhandari**
Integrated Data Binding API

### CODE REVIEW

File: app/src/main/java/org/gdghyderabad/sherlock/listener/RecyclerItemClickListener.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import android.view.View;
+
+public class RecyclerItemClickListener implements RecyclerView.OnItemTouchListener {
+    private OnItemClickListener mListener;
+
+    public interface OnItemClickListener {
+        void onItemClick(View view, int position, float x, float y);
+    }
+
+    GestureDetector mGestureDetector;
```
Comment (mustafa01ali) [inline app/src/main/java/org/gdghyderabad/sherlock/listener/RecyclerItemClickListener.java]: Make this private

File: app/src/main/res/layout/row_search_result.xml [hunk 2]
```diff
@@ -1,53 +1,70 @@
...
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <import type="org.gdghyderabad.sherlock.model.Book" />
+
+        <variable
+            name="book"
+            type="Book" />
```
Comment (mustafa01ali) [inline app/src/main/res/layout/row_search_result.xml:11]: Shouldn't this be com.foo.Book?

File: app/src/main/java/org/gdghyderabad/sherlock/listener/RecyclerItemClickListener.java (hunk 1 above)
Comment (phanirajabhandari) [inline app/src/main/java/org/gdghyderabad/sherlock/listener/RecyclerItemClickListener.java]: changed to private

File: app/src/main/res/layout/row_search_result.xml (hunk 2 above)
Comment (phanirajabhandari) [inline app/src/main/res/layout/row_search_result.xml:11]: updated the package name

File: app/src/main/java/com/mutualmobile/example/databinding/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.mutualmobile.example.databinding.model;
+
+/**
+ * Created by Mustafa Ali on 11/03/15.
+ */
+public class Book {
+    public VolumeInfo volumeInfo;
```
Comment (mustafa01ali) [inline app/src/main/java/com/mutualmobile/example/databinding/model/Book.java]: Make this private

**Pull request merged**